how-to/backups/install-gitolite.md

```diff
@@ -1,88 +1,148 @@
 ```
 
-Now we want to let gitolite know about the repository administrator's public SSH key. This assumes that the current user is the repository administrator. If you have not yet configured an SSH key, refer to the section on {ref}`SSH keys in our OpenSSH guide <openssh-server>`.
+To access the config repository, we now add the administrator's SSH public key.
+If you have not yet configured an SSH key, refer to the section on {ref}`SSH keys in our OpenSSH guide <openssh-server>`.
 
 ```bash
-cp ~/.ssh/id_rsa.pub /tmp/$(whoami).pub
+# assuming you are the administrator
+# if you have an rsa key, use id_rsa.pub.
+cp ~/.ssh/id_ed25519.pub /tmp/admin.pub
 ```
 
-Let's switch to the git user and import the administrator's key into gitolite.
+As the `git` user, let's import the administrator's key into gitolite (it will get the `admin` username due to that filename).
 
 ```bash
-sudo su - git
-gl-setup /tmp/*.pub
+sudo -i -u git gitolite setup -pk /tmp/admin.pub
 ```
 
-Gitolite will allow you to make initial changes to its configuration file during the setup process. You can now clone and modify the gitolite configuration repository from your administrator user (the user whose public SSH key you imported). Switch back to that user, then clone the configuration repository:
+What this creates:
+- management repo in `~git/repositories/gitolite-admin.git`
+- a global config in `~git/.gitolite.rc`
+- `~git/projects.list` as repo overview
+- `~git/.ssh/authorized_keys` with `command=` to force gitolite over `ssh`
+  - and for each user, their `ssh` public key
+
+To try if the setup worked, try `ssh` as the user owning the admin key we just added:
 
 ```bash
-exit
-git clone git@$IP_ADDRESS:gitolite-admin.git
-cd gitolite-admin
+# prints the gitlite repo overview
+ssh git@yourserver
+```
 ```
+hello admin, this is git@your-gitolite-server running gitolite3
 
-The `gitolite-admin` contains two subdirectories:
+ R W	gitolite-admin
+ R W	testing
+```
 
-- **`conf`** : contains the configuration files
-- **`keydir`** : contains the list of user's public SSH keys
 
 ## Managing gitolite users and repositories
 
-Adding a new user to gitolite is simple: just obtain their public SSH key and add it to the `keydir` directory as `$DESIRED_USER_NAME.pub`. Note that the gitolite usernames don't have to match the system usernames - they are only used in the gitolite configuration file to manage access control. 
+To configure gitolite users, repositories and permissions, clone the configuration repository:
+
+```bash
+# you can use your ip-address, hostname, or if working just locally use localhost
+git clone git@$yourserver:gitolite-admin.git
+
+cd gitolite-admin
+```
 
-Similarly, users are deleted by deleting their public key files. After each change, do not forget to commit the changes to git, and push the changes back to the server with:
+To apply configuration change, **commit them** in the repo and push the changes back to the server with:
```

Comment:
```suggestion
To apply configuration change, **commit them** in the repo and **push the changes** back to the server with:
```

otherwise this loses the emphasis on "push it back"